docs/internal-documentation/maintenance/Garbage-Collection-Testing.md

```diff
@@ -0,0 +1,122 @@
+# Testing instructions for the Garbage Collection feature of the Zonemaster Backend module
+
+## Introduction
+The purpose of this instruction is to serve as a notice for manual testing of the new garbage collection feature at release time.
+
+## Testing the unfinished tests garbage collection feature
+
+1. Ensure that the database has the required aditionnal columns for this feature:
+```
+SELECT nb_retries FROM test_results LIMIT 0;
+```
+Should return:
```

Comment:
>>>
For PostgreSQL write:
```
echo "SELECT nb_retries FROM test_results LIMIT 0;" | sudo -u postgres psql zonemaster
```
<<<

Please also give the equivalent for MySQL/MariaDB.

Reply:
Fixed in the commit 65f5257